_docs/developer/GSOC2018_GaganKumar.md

```diff
@@ -3,4 +3,131 @@ title: Discussion Forum by Gagan Kumar
+
+This project helped to inculcate a spirit of teamwork in me and also instilled the coordination, cooperation and management required to work in a team.
+
+I also absorbed the importance of code reviews/feedbacks by others as they might have a completely different angle to view the problem.
+
+#### [Discussion Forum](http://submitty.org/instructor/forum)
+
+Submitty has a student-faculty portal named **Discussion Forum**. It can be used for asking queries, announcements, notes sharing, etc.
+
+###### Contribution
+
+ [28 commits](https://github.com/Submitty/Submitty/commits?author=scopeInfinity) ![Commits](data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGYAAAAPCAYAAAAWJLrwAAAABHNCSVQICAgIfAhkiAAAABl0RVh0U29mdHdhcmUAZ25vbWUtc2NyZWVuc2hvdO8Dvz4AAAXeSURBVFiF7ZhfTFvnFcB/2GlsZAg2ZcZLQXEoae+2kjhiUs1Lh6ZK42nch6VEoHTOHgpTJ4VFlYK2ZGJP8UOzROqkoD2s6bSoXTMpZFI1S/OKUbQEbbA4JBsugcQDd2CHP+aP43sB++wBlz/GsJCmazTlJ13pft85537nfuf7zvnuzRMR4SlPHIYv24Gn5GZ9YPQ+2v52FIdfxfpxM2qoj/FcVno/3r5mlMBRnIFm1MF+4hmRNuvHc70Jq1/FGjhO61g0I4ni+2cbrsBRlMBR3Dc6CS5+gW+WRSr4W0aVKoZar2+upA0x5VEZslYx6FT59MIQaQBGmax9gU/Mq9eg8yckVgxjzLU1cMf8OlM5J+wRkBXm5HLvITlwq1fGlkSSD/4kDV2N0nxfl/Xo0nWzUQ7cuinTIiILN+VYd6N8P6qLyLCcuNoo9eFhSYrI2P135MCfT8r7CyLTkZNSeu1DGVgSEZmSy39vlBcHhmVTFoblj5Pj25flIHX9tNx9+biMNdTJnWPXNtV7cLpO7jR8KFpSJHXviozsqZPoDRGR2xI9UC/RgVxWUZluqJPwiXYJFx2RybGHdmtL1u0YZ/mPubCvGocRzPmvoBYmCCcXskKZIJRcRLHvxwrwjEJtYYJwMgEpC3V7j3NuTwVmwFHyCm7DNOEkhOdiWAurUIwANtzF5YwnRtE2WzHJbtrvhXLLt5LlwOD4Hl8NnMHqMm2hNcTcBzqW1kOYzGBwfpdnVZ2EbwjQSWkmjOZcdibM7e+xx+tmx0P681A+r94W4LLX8Jnv8fgVLsxVoBYXZJnYcNvsBMevL6c5vQ/fXBl1NhsYS6l9rhpnRlOL99FDBW4LKCVVEPcTWARSUXwTUdwlCjnf9XHjrMT8XweKsDhehsm52rPTVcZSMEKaWdLx+8y3qgw5qrijvM5Y52gmzRVhVuyP3eWNQdb9qFfPcgU7DftO4bFsNHLtfQO192c4P7ZgTSdw7D6Fd1eWUrKblts9uL9+hlojUNLEuV3HUbtfw2xYAEsTvrLSDc/29TdxOLacvWfS4PCfBUMF7d/8BUp4c1lr9vjbRZslrZkwrAlgntmEaLMIz5OvukmpP6S4s5y07+dEPG8x7fodzzo/57ibsVmOS85fk2NXG6UhMpUlmZL3ez3yrU8GMjVmWM72NEr9yKpecuYjqe9+U5qjq333wifkxb9+lKkxc9L1jzdlz81ry8/Ixcyv5eXegCS3K9uC5On6LWpMl0RK19eIxXePyGBDl6Q26MZl+jsvyb/eja/p80lkixqjnT8ig6XVMlhaLeHzIxva2azZMVECn97F4ahBMYLZUkOL/SK1k3fhuepVtdRdfDMWVEXJ1JgKVLsN70QIymsg4edw0I/7JS9tK2lwnp5YCKfjZKbGFFC7ez/m/luEqMH9BS267VHGTmcEPQw4lnsWghF2uMowaDGSIZ2drnKMAOjLaWwbedjkeYe9qg6YyLMWYWR9O5s1NWaB4MhZWoZDy0ffxRCdkzGUwjIAxqe66YzPg9GOKz+GbyJTuFNRAhMxnJYyYJSO2xcx7zu1JigABTgtNkITnx2/FwjG+onnV6zUoy+LxcAfmA3OAJUUqrtIeC+ha5AOX2KicxcFaiXEu7lf9wYxX4w0sBQ4z0zwIAW1Gyd0U8xF7HDY2eEoWj5EZLezyBNZ/fLXZv20Dlykc24azWDDZW+i42uvohjBd+M12ixnCL5QvqLnSy7bOYub6PjGqyjaJdx/uUBw3RDPoFZ18kFJiI7bv+TcTAJYwJxfQ/v+H6Hmb3cqt8+D1m8T6Ygg+vIKxQTGlveoOFfDvKeKSeUye9sqQRsi3vIWE53DpK3PU+B9G8fhSgzAos/LWOvv0cZ18pwHKfS+jaPODuOXGFF+SlIDdB1MJjC7+UroNxQ7Ht3ndYHZkvivaJk7REe57dFHexLp8fLv0A/Y7Xn8J6vPw0P/khlPV+Gx/58FBVjSDmJVn6ygwHZ2zFP+pzz9ifmE8h8CdpYpA39fOQAAAABJRU5ErkJggg==)
```

Comment:
In addition to the number of commits it might be nice to include your SLOC additions/removals.